[2024-08-05T09:20:17.889+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T09:20:17.911+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T21:00:00+00:00 [queued]>
[2024-08-05T09:20:17.922+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T21:00:00+00:00 [queued]>
[2024-08-05T09:20:17.923+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-05T09:20:17.939+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_air_pollution_data_Nairobi> on 2024-08-04 21:00:00+00:00
[2024-08-05T09:20:17.945+0300] {standard_task_runner.py:64} INFO - Started process 20924 to run task
[2024-08-05T09:20:17.949+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'air_pollution_data_dag', 'fetch_air_pollution_data_Nairobi', 'scheduled__2024-08-04T21:00:00+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_data_dag.py', '--cfg-path', '/tmp/tmpi9r84qdj']
[2024-08-05T09:20:17.950+0300] {standard_task_runner.py:91} INFO - Job 153: Subtask fetch_air_pollution_data_Nairobi
[2024-08-05T09:20:17.988+0300] {task_command.py:426} INFO - Running <TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T21:00:00+00:00 [running]> on host std21092
[2024-08-05T09:20:18.067+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='air_pollution_data_dag' AIRFLOW_CTX_TASK_ID='fetch_air_pollution_data_Nairobi' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T21:00:00+00:00'
[2024-08-05T09:20:18.068+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T09:20:19.266+0300] {logging_mixin.py:188} INFO - Erreur lors de la récupération des données pour Nairobi
[2024-08-05T09:20:19.266+0300] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T09:20:19.267+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T09:20:19.273+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=air_pollution_data_dag, task_id=fetch_air_pollution_data_Nairobi, run_id=scheduled__2024-08-04T21:00:00+00:00, execution_date=20240804T210000, start_date=20240805T062017, end_date=20240805T062019
[2024-08-05T09:20:19.324+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T09:20:19.345+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T09:20:19.349+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
